cnames_active.js

```diff
@@ -3418,6 +3418,7 @@ var cnames_active = {
   "vanilla": "haeresis.github.io/vanilla-js-dom",
   "vanilla-picker": "sphinxxxx.github.io/vanilla-picker",
   "vanilla-swr": "droyson.github.io/vanilla-swr",
+  "vanillaboot": "github.com/eyallior/vanillaboot",
```

Comment:
```suggestion
  "vanillaboot": "eyallior.github.io/vanillaboot",
```
As the CI error notes, you'll want to target the GitHub pages hostname (and path), not the repository itself

Reply:
I prefer the repo, if it's ok

Reply:
The CNAME based system we provide can only be used for the Github pages and not redirect to the repo. So please make the necessary change or we can't process your request.